tests/bdd/bdd_test.go

```diff
@@ -0,0 +1,240 @@
+package bdd
+
+import (
+	"fmt"
+	"os/exec"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	appv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	scheme "k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/client-go/tools/clientcmd"
+
+	v1alpha1 "github.com/litmuschaos/chaos-operator/pkg/apis"
+	chaosEngineV1alpha1 "github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
+	clientV1alpha1 "github.com/litmuschaos/chaos-operator/pkg/client/clientset/versioned/typed/litmuschaos/v1alpha1"
+)
+
+var kubeconfig = "/home/circleci/.kube/config"
+var config, _ = clientcmd.BuildConfigFromFlags("", kubeconfig)
+var client, _ = kubernetes.NewForConfig(config)
+var clientSet, _ = clientV1alpha1.NewForConfig(config)
+
+func TestChaos(t *testing.T) {
+
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "BDD test")
+}
+
+var _ = BeforeSuite(func() {
+
+	err := v1alpha1.AddToScheme(scheme.Scheme)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	//Creating chaosEngine Crd
+	By("creating chaosengine crd")
+	err = exec.Command("kubectl", "create", "-f", "../../deploy/crds/chaosengine_crd.yaml").Run()
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	//Creating chaosExperiments Crd
+	By("creating chaosexperiment crd")
+	err = exec.Command("kubectl", "create", "-f", "../../deploy/crds/chaosexperiment_crd.yaml").Run()
```

Comment:
In place of defining the separate command for all crd, You can use directly https://github.com/litmuschaos/chaos-operator/blob/master/deploy/chaos_crds.yaml